netkan/netkan/mirrorer.py

```diff
@@ -17,8 +17,10 @@ class Mirrorer:
     EPOCH_ID_REGEXP = re.compile('-[0-9]+-')
     EPOCH_TITLE_REGEXP = re.compile(' - [0-9]+:')
 
-    def __init__(self, ckan_meta_repo, ia_access, ia_secret, ia_collection):
+    def __init__(self, ckan_meta_repo, queue, ia_access, ia_secret, ia_collection):
```

Comment:
`cli.mirror_purge_epochs` doesn't pass queue.

Reply:
Yep, I only changed what was needed to get it running. That's going to need changes for the CLI as well, so will be addressed when I rebase that work

Reply:
But this means `cli.mirror_purge_epochs` is broken. Shouldn't that PR be rebased on working code?

Reply:
I'm comfortable with it remaining broken. It's going to need refactoring to fit anyway. In truth it probably belonged as a code snippet in a proposal about handling epochs and the archive better. But I wasn't going to hold up this PR for something that will be a manually run utility.

This PR is to line up the repo with production, any changes beyond that don't really fit the scope.

Reply:
OK, will create an issue instead.